os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -404,14 +406,13 @@ static void up_dumpstate(void)
  * Name: _up_assert
  ****************************************************************************/
 
-static void _up_assert(int errorcode) noreturn_function;
+static void _up_assert(int errorcode);
```

Comment:
I think this line is not required since we removed the noreturn_function attribute.

Reply:
changed